src/devices/Nmea0183/QuerySentence.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Iot.Device.Nmea0183
+{
+    /// <summary>
+    /// Represents NMEA0183 query sentence
+    /// </summary>
+    public class QuerySentence
+    {
+        public TalkerId RequesterId { get; private set; }
+        public TalkerId DeviceId { get; private set; }
+        public SentenceId RequestedSentence { get; private set; }
+
+        public override string ToString() => $"${RequesterId}{DeviceId}Q,{RequestedSentence}";
+
+        /// <summary>
+        /// Reads NMEA0183 query sentence from provided string
+        /// </summary>
+        /// <param name="sentence">NMEA0183 query sentence</param>
+        /// <returns>QuerySentence instance</returns>
+        /// <remarks><paramref name="sentence"/> does not include new line characters</remarks>
+        public static QuerySentence FromSentenceString(string sentence)
+        {
+            // http://www.tronico.fi/OH6NT/docs/NMEA0183.pdf
+            // page 3
+            // $XXYYY ...
```

Comment:
Should we say here that X means requester and Y represents deviceId? Or perhaps in the doc of this method we should mention how to create query sentences.